test/system/roles/public_health/dashboard/dashboard.rb

```diff
@@ -146,11 +146,12 @@ def select_monitorees_to_import(rejects, accept_duplicates)
   def select_monitorees_for_bulk_edit(workflow, tab, patient_labels)
     click_on 'Isolation Monitoring' if workflow == :isolation
     @@system_test_utils.go_to_tab(tab)
+    @@system_test_utils.go_to_tab(tab)
```

Comment:
As discussed with @timwongj I was experiencing some behavior where the datatables tab would not properly load unless clicked on twice.